t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -128,16 +128,6 @@ sub start_driver {
     return $_driver;
 }
 
-sub make_screenshot($) {
```

Comment:
This function can be useful for debugging in the headless CI environment. But yes, maybe here is not the best place for it.